src/item.js

```diff
@@ -18,6 +20,12 @@ export default class Item {
         this.element.removeEventListener(event, fn);
     }
 
+    setPreviousPosition() {
+        const oldTransform= this.element.style.transform.replace(/[(px,)]/g, ' ').split(/\s+/g)
```

Comment:
I'm 100% sure that you can store coordinates in the class and avoid string parsing by regexp. It's an antipattern to use regexp for parsing styles and code.

Reply:
fixed